schema/sorts/artwork_sorts.js

```diff
@@ -76,6 +76,12 @@ export default {
       PARTNER_UPDATED_AT_DESC: {
         value: "-partner_updated_at",
       },
+      POSITION_ASC: {
+        value: "position",
```

Comment:
This is potentially misleading since `'position'` is a valid sort only for the `/collection/:id/artworks` endpoint, but is not part of the generic `Artwork::SORT` [options](https://github.com/artsy/gravity/blob/master/app/models/domain/artwork.rb#L130-L143). I think it would be better to have a separate file to include these, or at least merge them in explicitly when defining the `sort: ` option above.